sonic_platform_base/chassis_base.py

```diff
@@ -118,6 +118,26 @@ def get_reboot_cause(self):
         """
         raise NotImplementedError
 
+    def get_controlcard_slot(self):
+        """
+        Retrieves the slot of the control card of the modular chassis
+
+        Returns:
+            An integer, the slot identifier of the control card of the
+            modular-chassis.
+        """
+        return 0
+
+    def get_my_slot(self):
+        """
+        Retrieves the slot of this card of the modular chassis
+
+        Returns:
+            An integer, the slot identifier of this card of the
+            modular-chassis.
+        """
+        return 0
```

Comment:
why the return value is zero?

Reply:
change to 'raise NotImplemented'. Add in documentation.

Reply:
return NotAvailable to make it backward compatible.

Reply:
Added back raise NotImplemented. Will catch exception and handle it when used.